examples/http/tests/test_http.py

```diff
@@ -19,11 +19,10 @@
 
 
 class TestHttpExample(unittest.TestCase):
-    @classmethod
-    def setup_class(cls):
+    def setUp(self):
```

Comment:
Were these changes required as part of this PR?

Reply:
Actually not, reverted.